src/core/withExperiment.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+import * as React from 'react';
+import invariant from 'invariant';
+import cookie from 'react-cookie';
+
+export type ExposedWithExperimentProps = {|
```

Comment:
Sorry for having given you the wrong name. I double checked and we already use another name pattern that is: `<HOC>InjectedProps`, so `WithExperimentInjectedProps`.